contrib/znc/files/znc

```diff
@@ -0,0 +1,5 @@
+type = process
+command = /usr/bin/znc
+log-type = buffer
```

Comment:
what does it normally log? buffer is super small and overflows really fast, so if it logs any informational non-error messages this should be log-file instead most likely

Reply:
it doesnt log a lot.

```
Checking for list of available modules... 
Opening config [/jbx/.znc/configs/znc.conf]... 
Loading global module [corecaps]... 
Loading global module [webadmin]... 
Binding to port [1040]... 
Loading user [jaba]
Loading user module [chansaver]... 
Loading user module [controlpanel]... 
Forking into the background... [pid: 24658]
ZNC 1.9.1 - https://znc.in
```

Reply:
this logs enough to show that it doesn't work at all; the 'fork into background' makes dinit lose track of it instantly and spawn a new one:  

```
    26258 doas dinitctl start znc
      26259 /usr/bin/unix_chkpwd demon chkexpiry
      26260 dinitctl start znc
26261 /usr/bin/znc -d /var/lib/znc
26264 /usr/bin/znc -d /var/lib/znc
26267 /usr/bin/znc -d /var/lib/znc
26269 /usr/bin/znc -d /var/lib/znc
26270 /usr/bin/znc -d /var/lib/znc
26271 /usr/bin/znc -d /var/lib/znc
```

and the final state is stopped because all of them 'died', even if it's running in the end

Reply:
you need to pass --foreground to znc

Reply:
after that, yeah seems to work